server/status.go

```diff
@@ -137,10 +138,14 @@ func (s *statusServer) handleGossipStatus(w http.ResponseWriter, r *http.Request
 // handleLocalStatus handles GET requests for local-node status.
 func (s *statusServer) handleLocalStatus(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
 	local := struct {
+		Address   net.Addr       `json:"address"`
```

Comment:
`net.Addr` is an interface; we can't assume that all implementations of the interface will have the proper json tags. If we can guarantee that we have an UnresolvedAddr here then use that type explicitly; otherwise we'll need to define a local struct for the address and pass addr.Network() and addr.String() explicitly.